response_operations_ui/templates/create-collection-exercise.html

```diff
@@ -3,8 +3,10 @@
 {% from "components/metadata/_macro.njk" import onsMetadata %}
 {% from "components/fieldset/_macro.njk" import onsFieldset %}
 {% from "components/input/_macro.njk" import onsInput %}
+{% from "components/accordion/_macro.njk" import onsAccordion %}
 
 {% set page_title = 'Create collection exercise' %}
+{% set previous_period = request.args.get('previous_period') %}
```

Comment:
hmm, we've only done this in one place so I'm inclined so say to use template variables in the python code instead of doing request.args.get here... but then again I don't know if it's actually bad practice or not?  I think keeping as much code out of the templates is probably right?